Net31Solution/SensorApp/CalibrationValueGenerator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace SensorApp
+{
+    internal class CalibrationValueGenerator : IValueGenerator
+    {
+        private static int orderNumber = 0;
+
+        public int GetMeasuredValue()
+        {
+            return Interlocked.Increment(ref orderNumber);
```

Comment:
It is not just increasing sequence, it should grow in time. 
Let say CalibrationValueGenerator  has been created 10 seconds ago should return 10 when it is called at first time.